### ISSUE TITLE
Is docs.https.dev no longer live?

### ISSUE THREAD

**Issue by mcpherrinm** [https-dev/docs#6]
The hostname docs.https.dev no longer resolves, and https.dev seems to have a redirect loop.

Looks like Archive.org last captured in January 2021:
https://web.archive.org/web/20210126190116/https://docs.https.dev/

**Comment by mholt** [comment]
Yeah, Google charged over $800/yr for the domain, so I let it expire. Sorry. I need to update this repo.

**Comment by mcpherrinm** [comment]
Thanks for the update— and that’s pretty unreasonably expensive for a domain!
